### PR TITLE
fix(slide dimension) keep fitToWidth property on reloaded presentation

### LINKED ISSUE #14107: Quick switch between External Video and Presentation
Hi there, we use external video in class often and find it a bit annoying to switch between video and the presentatioin (pdf), since the document would reload dagain when we close the video to show the presentation again.  It is especially inconvenient when we ask students to answer questions based on video details, which requires us to switch between video and doc way too often with many reloads and trying to find the previous position again and again, quite time consuming. 

**Describe the solution you'd like**
Can we have a quick swtich option that allows us to show video or doc without loading again? 

Please find the mockup attached; I hope I have made it clear, thanks a lot.
quick-switch

### LINKED ISSUE #14154: fitToWidth properties not kept after video/screen sharing
**Describe the bug**
After quitting the video sharing or screen sharing, the presentation discards the fitToWidth information so the focus position within the slide changes when the presenter uses portrait slide and use it in the fit-to-width mode.

This phenomenon was found by @yayaxueyu2021. The report is #14107.

**To Reproduce**
1. upload a portrait mode presentation
2. hit "fit to width" button
3. zoom and pan somewhere in the slide
4. start screen sharing or external video sharing
5. quit sharing
6. now the position is not the same as before, the presentation stops fitting to the width

**Expected behavior**
We stay at the same position in the slide after sharing screen or video.

**Actual behavior**
We will be in a different place within the slide and the slide is not "fit to width" anymore

**Screenshots**
https://user-images.githubusercontent.com/84652647/150315681-741fda87-b79b-416c-aadb-df2877e5b912.mp4

### PR DESCRIPTION
**Pull request by hiroshisuga**
### What does this PR do?

Keeps the fitToWidth property in a reloaded presentation (only) for the presenter, so that a portrait slide that is fit to the width can maintain its zoom rate and focal position even after [minimising slide]/[sharing screen]/[sharing external video]/[going out and in the room]/[switching presentations].

This is another implementation of #14178, which has been closed.

### Closes Issue(s)

Closes #14107, #14154

### Motivation

It has been uncomfortable that a portrait slide doesn't keep its focal point and the scaling due to the missing fitToWidth property after reloading a presentation.

### More

Before the change (at test24.bigbluebutton.org):
before

After the change:
after

#14178 adds the information to each slide, whereas this PR adds it to the meeting. This way we can solve the problem in a much simpler way. 

This PR adds the fitToWidth information to the mongoDB of the presenter. So if the presenter changes, the information will be reset as default (false).

### CODE REVIEW

**Comment by hiroshisuga** [comment]
No problem found in recording and playback.

File: bigbluebutton-html5/imports/api/meetings/server/methods/setFitToWidth.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      meetingId,
+    };
+
+    const modifier = {
+      $set: {
+        fitToWidth,
+      }
+    }
+
+    const { insertedId } = Meetings.upsert(selector, modifier);
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/meetings/server/methods/setFitToWidth.js:24]: Note - here a new `Meeting` property is being added in MongoDB and it does not reach akka-apps. I'll need to think about this some more.

**Comment by antobinary** [comment]
Thank you for your pull request, @hiroshisuga ! We'll be reviewing it soon!

File: bigbluebutton-html5/imports/api/meetings/server/methods/setFitToWidth.js (hunk 1 above)
Comment (hiroshisuga) [reply bigbluebutton-html5/imports/api/meetings/server/methods/setFitToWidth.js:24]: Correct. I did it so on purpose, wondering (still) what is the best way to implement. 
We can also send this property to akka-apps and propagate it to other users/viewers, which can potentially become presenters. Actually I tried to do so initially, but then I realised that we do not need to send it to other viewers; we just want BBB to "remember" it just for the presenter. I also felt that the fitToWidth property is rather a personal setting than a meeting-wide.
Of course this PR is just an idea, you can decide the design. Thanks for the comment!

Comment (hiroshisuga) [reply bigbluebutton-html5/imports/api/meetings/server/methods/setFitToWidth.js:24]: Note: just to remember the fitToWidth property during the session, we can use the meteor/session. I don't know which is better.

**Pull request closed without merging**